_posts/en/newsletters/2023-01-04-newsletter.md

```diff
@@ -0,0 +1,200 @@
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #26265][] POLICY: Relax MIN_STANDARD_TX_NONWITNESS_SIZE to 65 non-witness bytes FIXME:glozow
+(maybe link to
+https://bitcoinops.org/en/newsletters/2022/10/19/#minimum-relayable-transaction-size
+and/or
+https://bitcoinops.org/en/newsletters/2022/11/09/#bitcoin-core-pr-review-club )
+
+- [Bitcoin Core #21576][] allows wallets using an external signer to fee bump
+  using [opt-in RBF][topic rbf] in the GUI and when using the `bumpfee` RPC.
+
+- [Bitcoin Core #24865][] allows a wallet backup to be restored on a
+  node that has been pruned of older blocks as long as the node still
+  has all of the blocks produced after the wallet was created.  The
+  blocks are needed so that Bitcoin Core can scan them for any
+  transactions affecting the wallet's balance.  Bitcoin Core is able to
+  determine the age of the wallet because its backup contains the date
+  the wallet was created.
+
+- [Bitcoin Core #23319][] updates the `getrawtransaction` RPC to provide
+  additional information if the `verbose` parameter is set to `2`.  The
+  additional information includes the fee the transaction paid and
+  information about each of the outputs from previous transactions
+  ("prevouts") which are spent by being used as inputs to this
+  transaction.  See [Newsletter #172][news172 prevout] for details about
+  the method used to retrieve the information.
+
+- [Bitcoin Core #26628][] begins rejecting RPC requests that include the
+  same parameter name multiple times.  Previously, the daemon treated a
+  request with repeated parameters as if it only had the last of the repeated
+  parameters, e.g. `{"foo"="bar", "foo"="baz"}` was treated as
+  `{"foo"="baz"}`.  Now the request will fail.  When using `bitcoin-cli`
+  with named parameters, the behavior is unchanged: repeated parameters
+  will not be rejected but only the last of the repeats will be sent.
+
+- [Eclair #2464][] adds the ability to trigger an event when a remote
+  peer becomes ready to process payments.  This is especially useful in
+  the context of [async payments][topic async payments] where the local
+  node temporarily holds a payment for a remote peer, waits for the peer
+  to connect (or reconnect), and delivers the payment.
+
+- [Eclair #2482][] allows sending payments using [blinded routes][topic
+  rv routing], which are paths whose last several hops are chosen
+  by the receiver.  The receiver uses onion encryption to obfuscate the
+  hop details and provides the encrypted data to the spender, along with
```

Comment:
```suggestion
  hop details and provide the encrypted data to the spender, along with
```

Reply:
I'm not sure about your suggestion, @bitschmidty; I think the subject of the verb "provides" is the receiver (singular). Maybe it seems like "provide" is correct because "details" is plural?